index.html

```diff
@@ -19,10 +19,30 @@
       color: #4c4c4c;
       white-space: pre;
     }
+
+		.btn {
```

Comment:
Nitpick: Can you please adjust the indentation to match the current style. Thank!